app/models/v2/test_result.rb

```diff
@@ -8,7 +8,8 @@ class TestResult < ApplicationRecord
     self.primary_key = :id
 
     belongs_to :system, optional: true
-    belongs_to :tailoring
+    belongs_to :tailoring, class_name: 'V2::Tailoring'
```

Comment:
The `:system` above will need a `class_name` and the `has_one` calls beneath too...